manager/api/es/src/main/java/io/apiman/manager/api/es/EsStorage.java

```diff
@@ -1688,9 +1748,9 @@ public int getMaxPolicyOrderIndex(String organizationId, String entityId, String
         SearchSourceBuilder builder = new SearchSourceBuilder()
                 .fetchSource(fields, null)
                 .query(qb)
-                .sort(new FieldSortBuilder("name").order(SortOrder.ASC))
+                .sort(new FieldSortBuilder("name.keyword").order(SortOrder.ASC))
```

Comment:
Are these okay or was this a test?

Reply:
Yeah, this is the final thing. Because a sort operation is being done here, we need to use the `.keyword` multifield (it's in the schema, so we can guarantee it will be there).